hw/top_darjeeling/top_darjeeling.core

```diff
@@ -90,7 +90,7 @@ filesets:
       - lowrisc:lint:common
       - lowrisc:lint:comportable
     files:
-      - lint/top_darjeeling.vbl
```

Comment:
That line should be reverted. CI greps for `*.vbl` in the repo to build a global waiver:
 
```sh
echo "Concatenating Verible waivers"
find . -type f -name '*.vbl' -exec cat {} \; >> verible_waiver
```

Reply:
Ah, I must have made a mistake during a rebase. That line change was definitely unintentional.

Reply:
Fixed that up.